micro_dl/input/dataset.py

```diff
@@ -69,21 +93,35 @@ def _augment_image(self, input_image, aug_idx):
          5 - rot 270
         :return np.array image after transformation is applied
         """
+        # We need to flip over different dimensions depending on data format
```

Comment:
I believe at this stage, we are augmenting either a 2D array (y-x) or 3D array (z-y-x), no channel axis available yet?

Reply:
Yes, I'm holding off on dealing with channels for now. I'll clarify that it doesn't deal with channels in the docstring.
When you say axis 1-2 do you mean for channels first data?
Good point about flipping about z!

Reply:
Sorry about the confusion here, trying to separate z and channel axis! 1-2 here is w.r.to 3D array in z-y-x format.